js/components/characterData.js

```diff
@@ -0,0 +1,56 @@
+export const characterData = [
```

Comment:
**nitpick:** This is not a component but a dumb data - it should not be placed here

Reply:
Could you elaborate more? How else should I store the character data?